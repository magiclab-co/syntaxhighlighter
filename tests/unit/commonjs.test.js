import Brush from '../fixtures/test_brush_v4';

describe('unit/commonjs', () => {
  let html;

  describe('using a brush', () => {
    let brush;

    beforeAll(() => {
      brush = new Brush();
      html = brush.getHtml('hello foo bar world!', { gutter: false });
    });

    it('returns html', () => expect(html).toBeTruthy());
    it('renders content', () => expect(html).toMatchSnapshot());
    it('does not render gutter', () => expect(html).toMatchSnapshot());
  });
});
