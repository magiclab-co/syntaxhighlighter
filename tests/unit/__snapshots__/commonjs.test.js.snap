// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`unit/commonjs using a brush does not render gutter 1`] = `
"
      <div class=\\"syntaxhighlighter nogutter  \\">
        <table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
          
          <tbody>
            <tr>
              
              <td class=\\"code\\">
                <div class=\\"container\\"><div class=\\"line number1 index0 alt2\\"><code class=\\"keyword\\">hello</code> <code class=\\"plain\\">foo bar </code><code class=\\"keyword\\">world</code><code class=\\"plain\\">!</code></div></div>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    "
`;

exports[`unit/commonjs using a brush renders content 1`] = `
"
      <div class=\\"syntaxhighlighter nogutter  \\">
        <table border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
          
          <tbody>
            <tr>
              
              <td class=\\"code\\">
                <div class=\\"container\\"><div class=\\"line number1 index0 alt2\\"><code class=\\"keyword\\">hello</code> <code class=\\"plain\\">foo bar </code><code class=\\"keyword\\">world</code><code class=\\"plain\\">!</code></div></div>
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    "
`;
